lib/aprb.ex

```diff
@@ -8,7 +8,7 @@ defmodule Aprb do
 
     children = [
       supervisor(Aprb.Repo, []),
-      worker(Aprb.Service.AmqEventService, [%{topic: "conversations"}], id: :conversations),
+      worker(Aprb.Service.AmqEventService, [%{topic: "conversations", routing_key: "conversation.*"}], id: :conversations),
```

Comment:
What are you trying to exclude by specifying the `conversation.*` routing key here?